include/deal.II/differentiation/sd/symengine_optimizer.h

```diff
@@ -1477,16 +1477,15 @@ namespace Differentiation
                        OptimizationFlags::optimize_all);
 
       /**
-       * Copy constructor
+       * Copy constructor.
        *
-       * The @p copy_initialized flag, which is set to <code>true</code> by default,
-       * determines whether or not all of the optimized data is copied over from
-       * the @p other optimizer instance. Only with the flag set to <code>false</code>
-       * is it possible to re-optimize the data stored in this class with a
-       * different optimization scheme.
-       */
-      BatchOptimizer(const BatchOptimizer &other/*,
-                     const bool            copy_initialized = true*/);
```

Comment:
Note: No implementation change here: This just removes a commented out parameter that was a legacy of the prototype implementation.

Reply:
Oh, I had not even realized that the parameter was in reality non-existent :-)

Reply:
No worries :-) It was an unusual change to review. I shouldn't have been so sloppy in the first place.